### PR TITLE
Support filtering liveStreamScheduleEvents based on source entry id

### PR DESCRIPTION
**Pull request by yossipapi**

### CODE REVIEW

File: plugins/schedule/base/config/IndexSchema.xml [hunk 1]
```diff
@@ -32,6 +32,7 @@
 	        <field name="template_entry_categories_ids" indexName="template_entry_categories_ids" getter="templateEntryCategoriesIdsForIndex" type="string" matchable="yes" />
 	        <field name="resource_system_names" indexName="resource_system_names" getter="resourceSystemNamesForIndex" type="string" matchable="yes" />
 	        <field name="template_entry_id" indexName="template_entry_id" getter="templateEntryId" type="string" matchable="yes" />
+			<field name="source_entry_id" indexName="source_entry_id" getter="sourceEntryId" type="string" matchable="yes" />
```
Comment (inbal-ben-david) [inline plugins/schedule/base/config/IndexSchema.xml:35]: indentation

File: release-notes.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
 # Scorpius-19.9.0
+
+## Schedule event new filtering options ##
```
Comment (inbal-ben-david) [inline release-notes.md:3]: why under 19.9? its 3 versions ago

File: plugins/schedule/base/lib/model/ScheduleEvent.php [hunk 3]
```diff
@@ -350,6 +350,11 @@ public function getTemplateEntryId()
 		return null;
 	}
 
+	public function getSourceEntryId()
```
Comment (inbal-ben-david) [inline plugins/schedule/base/lib/model/ScheduleEvent.php:353]: why it's not like
plugins/schedule/base/lib/model/LiveStreamScheduleEvent.php:30
also, it wont conflict with the existing one defined here?
alpha/apps/kaltura/lib/scheduleEvent/ILiveStreamScheduleEvent.php:5

Comment (yossipapi) [reply plugins/schedule/base/lib/model/ScheduleEvent.php:353]: Its like this as the index contains more than 1 type of ScheduleEvent and not all of them have the source entryId

File: plugins/schedule/base/config/IndexSchema.xml (hunk 1 above)
Comment (yossipapi) [reply plugins/schedule/base/config/IndexSchema.xml:35]: Done

**Pull request merged**